cram_pr2/cram_pr2_shopping_demo/package.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>cram_pr2_shopping_demo</name>
+  <version>0.0.0</version>
+  <description>The cram_pr2_shopping_demo package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="jonas@todo.todo">jonas</maintainer>
+
+
+  <!-- One license tag required, multiple allowed, one license per tag -->
+  <!-- Commonly used license strings: -->
+  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <license>TODO</license>
+
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>cl_bullet</build_depend>
```

Comment:
For all the packages which have "build_depend", "build_export_depend", and "exec_depend" please simply use "<depend>package</depend>". Otherwise the dependencies are very difficult to read.